cli/internal/config/config_file_test.go

```diff
@@ -102,7 +147,7 @@ func TestReadUserConfigWhenMissing(t *testing.T) {
 }
 
 func TestWriteUserConfig(t *testing.T) {
-	fsys := afero.NewMemMapFs()
+	fsys := afero.NewOsFs()
```

Comment:
We, unfortunately, need to rip `afero` out.

XDG also only checks the OS file system, so it is pretty much nonsense to use an `afero` `MemMapFs` here.

Reply:
Yeah. Would've been nice.